src/documents/ResourceLoader.md

```diff
@@ -4,114 +4,261 @@ category: Infusion
 ---
 
 The Infusion Resource Loader ```fluid.resourceLoader``` is an [Infusion component](UnderstandingInfusionComponents.md)
-which offers a mechanism to load multiple physical files in one shot. These resources can be HTML templates, JSON
-objects or any other resources. On completion, the resource loader fires an ```onResourcesLoaded``` event with an
-argument of the populated structure with fetched resource text in the field "resourceText" for each entry. This
-populated structure can also be retrieved directly on the resource loader instance via the path ```resources```.
+which offers a mechanism to load multiple resources in one shot. Whilst these resources may require an asynchronous
+I/O operation, ``fluid.resourceLoader`` can arrange to load these before component instantiation has finished.
+Resources may be loaded over a variety of transports, such as HTTP/HTTPS, or
+directly from the filesystem if Infusion is running under `node.js`. These resources can be HTML templates, JSON objects
+or other kinds of static resources.
 
-The underlying implementation of ``fluid.resourceLoader`` is the low-level [``fluid.fetchResources``](FetchResources.md)
-API which in turn defers to [jQuery.ajax](http://api.jquery.com/jquery.ajax/).
+The resources to be loaded are configured in a free hash held in the top-level ``resources`` option of the component.
+Each member of ``resources`` is a structure named ``ResourceSpec`` specifying a resource to be loaded.  
 
-Although we show an example here of constructing the resource loader as a free-standing component, a more typical usage
-would be as a [subcomponent](SubcomponentDeclaration.md) in a wider component tree:
+### Minimal ``fluid.resourceLoader`` example
+
+Although we show an example here of constructing the resource loader as a free-standing component,
+a more typical usage would configure ``fluid.resourceLoader`` as a mixin to some
+[subcomponent](SubcomponentDeclaration.md) in a wider component tree:
 
 ```javascript
-var resourceLoader = fluid.resourceLoader([options]);
+var resourceLoader = fluid.resourceLoader({
+    resources: {
+        userRecord: {
+            url: "https://jsonplaceholder.typicode.com/users/1",
+            dataType: "json"
+        }
+    },
+    listeners: {
+        "onResourcesLoaded.log": {
+            funcName: "console.log",
```

Comment:
Does this work now? I feel like I've had issues because of the thisist nature of `console`.

Reply:
Yes, it works now. Not sure what has changed, but the behaviour of the modern browsers seems to have been standardised on this.